test/plugins/validation/oas3/operations.js

```diff
@@ -2,10 +2,19 @@ const expect = require('expect');
 const {
   validate
 } = require('../../../../src/plugins/validation/oas3/semantic-validators/operations');
-const config = require('../../../../src/.defaultsForValidator').defaults.oas3;
```

Comment:
Wait.. why have you removed this line? This is meant to handle the config so that you don't have to define it for every independent test. It should work for your use case